scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/server/SubmitEditServlet.java

```diff
@@ -246,4 +251,16 @@ private void upload(HttpServletRequest request, HttpServletResponse response) {
         }
     }
 
+    private String getTaskName(Node node) {
```

Comment:
This looks like the same method as the other class. Can you extract this method in an utility class and call it one time only?